manim/mobject/vector_field.py

```diff
@@ -606,6 +627,17 @@ def get_vector(self, point: np.ndarray):
             vect.set_color(self.pos_to_color(point))
         return vect
 
+    def fit_to_coordinate_system(self, coordinate_system: CoordinateSystem):
+        mat = np.eye(3)
+        for i, ax in enumerate(coordinate_system.get_axes()):
+            mat[i, :] = ax.get_unit_vector()
+        mat = mat.T
+        for vec in self.submobjects:
+            start_pos = vec.get_start()
+            center_to_start = vec.get_center() - start_pos
+            vec.move_to(coordinate_system.coords_to_point(*start_pos) + center_to_start)
+            vec.apply_matrix(mat, about_point=vec.get_start())
+
```

Comment:
I can see the point of this second implementation, but I am not sure I really like that there has to be a so different and more complex implementation when the `apply_function`-approach from the parent class also does the job. I guess it all depends on how difficult it is to change the style of the arrows when they are fitted to a CoordinateSystem. It should be possible to disable this weird scaling behavior somehow (but it might require iterating over all arrows and setting one of their `max_length_to_whatever` ratios "correctly".